pandas/io/gbq.py

```diff
@@ -106,13 +107,14 @@ class TableCreationError(PandasError, ValueError):
     """
     pass
 
-
 class GbqConnector(object):
 
-    def __init__(self, project_id, reauth=False):
+    def __init__(self, project_id, reauth=False, private_key_json_path=None):
         self.test_google_api_imports()
```

Comment:
thanks for feedback.

* which doc-string do you mean?
* this param is intended to contain only file path. Google console provides json files. Most convenient way to use them is store somewhere in local file system where pandas is running (eg. user home) and point that local path to this param. Putting its entire contents as parameter string-value will be impractical.